misc/vm_configs/scenarios/hybrid_rt/ehl-crb-b/misc_cfg.h

```diff
@@ -76,12 +76,17 @@
 #define VM3_VCPU_CLOS			{0U}
 #endif
 
+#define VM0_PASSTHROUGH_TPM
```

Comment:
This section would also be automatically generated by the `acrn-config` tool if someone were to re-generate the code files from the source XMLs, is that correct?

Reply:
Yes. We should make the xml configuration and configuration source code aligned when updating xmls or config tool.